### ISSUE TITLE
C-PORT: problems reloading a saved scenario

### ISSUE THREAD

**Issue by joellenb** [CEMPD/ctools#192]
I created and saved a scenario for Charleston, default hourly conditions, point sources only, pollutant acrolein. The scenario ran to completion and properly showed a map with no concentrations and an empty legend.

I then went back to my Scenarios list.The above scenario still appeared at the top of the list and was colored in red. However, when I load that scenario no buttons are active.  

![c-port_ptonly_acrolein_hrly_reloaded](https://cloud.githubusercontent.com/assets/10145737/11346836/984a7f9c-91ec-11e5-831a-0cbbbdc82fe3.PNG)
I even closed my Firefox browser, started cport (on treehug) again, and loaded this saved scenario. The screen and buttons again looked like the image shown above.

**Comment by cseppan** [comment]
Not so much a bug as an artifact of the system's design. Once a scenario is saved, there's no way to load sources back into it.

**Comment by joellenb** [comment]
Catherine,

What I am showing in this issue is behavior that did not exist.

I had sources in the scenario that I saved. When I reloaded it the same sources (point sources) that were in the saved scenario did not load; they did not load on the map and the point sources tile was not activated.

I do not believe that this is an artifact of the system’s design, which would not load any new sources or source types when a saved scenario is loaded but heretofore loaded the sources that were in the saved scenario. If the user cannot see anything about a saved scenario, then why have the ability to save it at all?

Jo Ellen
